src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static int addTaskItem(Task[] TaskList, String item, int numOfItems, String typeOfTask) throws Exception {
+
+        switch(typeOfTask){
+        case "todo":
+            if(item.equals("")){
+                throw new Exception("☹ OOPS!!! The description of a todo cannot be empty.");
+            }
+
+            TaskList[numOfItems] = new Todo(item,numOfItems);
+            break;
+
+        case "deadline":
+            if(item.equals("")){
+                throw new Exception("☹ OOPS!!! The description of a deadline cannot be empty.");
+            }
```

Comment:
You can SLAP out this validation and reuse it while processing the input. That would improve the SLAP in this method.